askama_parser/src/lib.rs

```diff
@@ -249,11 +248,75 @@ fn bool_lit(i: &str) -> ParseResult<'_> {
     alt((keyword("false"), keyword("true")))(i)
 }
 
+/// Underscore separateed digits of the given base, unless `must_start_with_digit` this may start
+/// with an underscore.
+fn separated_digits(radix: u32, must_start_with_digit: bool) -> impl Fn(&str) -> ParseResult<'_> {
```

Comment:
Please move this below `num_lit()` and rename `must_start_with_digit` to just `start`.

Reply:
done